### ISSUE TITLE
Citation-Abbreviations not being called

### ISSUE THREAD

**Issue by iandol** [htdebeer/paru#44]
Hi Huub, long time! Hope all is good with you. 

There is a small naming bug with V2 Pandoc options:

https://pandoc.org/MANUAL.html#citation-rendering

Should be `citation_abbreviations` but in https://github.com/htdebeer/paru/blob/master/lib/paru/pandoc_options_version_2.yaml#L116 it is `citation_abbreviation` and so we get the error:

```
The pandoc option 'citation_abbreviations' (with value '/Users/ian/.pandoc/cite-abbr.json') is not recognized by paru. This option is skipped.
```

I'll create a pull request...

**Comment by htdebeer** [comment]
Thanks for noticing and fixing the misnamed pandoc option in the options file! I've accepted and applied the pull request and your fix is delivered in release 0.3.1.2.

**Comment by iandol** [comment]
Very fast, thank you Huub!!!
